source/style-guide/style/tables.txt

```diff
@@ -123,10 +118,11 @@ can use footnotes.
 -  Place the footnote text at the end of the table, either in a final
    row that spans the entire table or under the last row in the table.
 
--  Use superscript numbers to indicate the footnotes in the cells to
-   which they apply. If numbers might be confusing (for example,
-   because the text in the cells are numerical values), use lowercase
-   letters instead.
+-  Use `RST markup 
+   <https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#footnotes>`_
+   to indicate the footnotes in the cells to which they apply. If
+   numbers might be confusing (for example, because the text in the
+   cells are numerical values), use symbols instead.
```

Comment:
```suggestion
-  Use :rst:`RST markup </restructuredtext.html#footnotes>`_
   to set the footnotes in the cells to which they apply. If
   numbers might be confusing (because maybe the text in the
   cells are numerical values), use symbols instead.
```